safe-global/safe-react-apps
# feat(safe-claiming-app): Claiming app change formula

## What it solves
Resolves #611 

## How this PR fixes it
- Excludes unredeemed allocations after the deadline was reached
- Governance voting power widget also excludes unredeemed allocations after the deadline
- refactors `useAirdropFile` and `useVestingStatus` hook into one `useSafeTokenAllocation`

## How to test it
On Görli the redeem deadline was already reached
- check a safe that did not redeem on görli
  - The claiming app / voting power widget should not include any allocations on görli
- check a safe thatz did not redeem on mainnet
  - The allocation should still be included
